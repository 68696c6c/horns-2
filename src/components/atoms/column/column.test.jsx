/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Column from '.'

describe('Column', () => {
  it('should render as default', () => {
    const { container } = render(<Column>example</Column>)
    expect(container).toMatchSnapshot()
  })
})
