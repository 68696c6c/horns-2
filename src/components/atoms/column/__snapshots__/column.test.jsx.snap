// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column should render as default 1`] = `
<div>
  <div
    class="emotion-0"
    span="1"
  >
    example
  </div>
</div>
`;
